package.json

```diff
@@ -8,7 +8,7 @@
   "typings": "typings/index.d.ts",
   "scripts": {
     "add-contributor": "kcd-scripts contributors add",
-    "build": "kcd-scripts build --bundle --p-react",
+    "build": "kcd-scripts build --bundle --p-react --no-clean",
```

Comment:
temp fix, this needs a fix in kcd-scripts - going to report an issue in a moment